.travis/archlinux/Dockerfile.deps.tmpl

```
@@ -12,4 +12,5 @@ RUN pacman -Syu --needed --noconfirm \
 	libyaml \
 	meson \
 	python-gobject \
+	python2-babel \
```

You want the python3 version here:

```suggestion
	python-babel \
```